style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
```

Comment:
Maybe `Use ES6 classes` would be a better cover all since it includes `function Cat() {}` and etc.